src/System.Management.Automation/FormatAndOutput/common/Utilities/MshObjectUtil.cs

```diff
@@ -208,8 +208,9 @@ private static string GetObjectName(object x, PSPropertyExpressionFactory expres
         /// <param name="expressionFactory">Expression factory to create PSPropertyExpression.</param>
         /// <param name="enumerationLimit">Limit on IEnumerable enumeration.</param>
         /// <param name="formatErrorObject">Stores errors during string conversion.</param>
+        /// <param name="formatFloat">Determine if to format floating point numbers using current culture.</param>
```

Comment:
Is this (name and definition) correct for decimals?

Reply:
I believe technically doubles and decimals are floating point numbers